rc/.eslintrc.json

```diff
@@ -247,14 +267,23 @@
         ],
         "new-parens": 2,
         "newline-after-var": 0,
-        "object-curly-spacing": [0, "never"],
+        "object-curly-spacing": [
+            2,
+            "never"
```

Comment:
This is different than what was in lint-trap and now break linting in applications which all do `{foo: 'bar'}`. Can we revert this?